src/Action/Iterator.php

```diff
@@ -110,9 +110,9 @@ public function order($fields)
 
         // prepare arguments for array_multisort()
         $args = [];
-        foreach ($fields as list($field, $desc)) {
+        foreach ($fields as list($field, $direction)) {
             $args[] = array_column($data, $field);
-            $args[] = $desc ? SORT_DESC : SORT_ASC;
+            $args[] = strtolower($direction) === 'desc' ? SORT_DESC : SORT_ASC;
```

Comment:
Yes, this is slighly questionable. I discovered some bugs because of this (like 'asc' was passed and evaluated to true), but with strict types (bug need to be enable on calling side) it might be worth to isolate this change into another PR. Wdyt?